metrics/mypy_high_water_mark

```diff
@@ -1 +1 @@
-90.6000
+89.7400
```

Comment:
I'm not sure why the coverage had to go down, to me it looks like everything is appropriately type hinted?